verilog/analysis/verilog_linter.cc

```diff
@@ -205,28 +209,53 @@ std::vector<LintRuleStatus> VerilogLinter::ReportStatus(
   return statuses;
 }
 
-LinterConfiguration LinterConfigurationFromFlags() {
-  LinterConfiguration config;
-
-  // Turn on default ruleset.
-  const auto& ruleset = absl::GetFlag(FLAGS_ruleset);
-  config.UseRuleSet(ruleset);
+const std::string linter_config_file = ".rules.verible_lint";
 
+bool AppendLinterConfigurationFromFile(
+    LinterConfiguration &config, std::string config_filename) {
   // Read local configuration file
   std::string content;
-  if (verible::file::GetContents(absl::GetFlag(FLAGS_rules_config), &content)) {
+  if (verible::file::GetContents(config_filename, &content)) {
```

Comment:
Note that this access is relative to the current working directory.  Need to make sure behavior is consistent with documentation.

Reply:
I think this is consistent with what all the tools do when providing a path to file. This can either be relative to current working directory or an absolute path.
Do you think the help message should be saying that explicitly?

Reply:
Hmm, this seems ok here.  I might have been thinking elsewhere around the upward file search.